package.json

```diff
@@ -6,33 +6,33 @@
   "bin": "lib/index.js",
   "bugs": "https://github.com/felipecrs/bindl/issues",
   "dependencies": {
-    "@felipecrs/decompress-tarxz": "^5.0.2",
-    "@xhmikosr/decompress-tar": "^8.0.1",
-    "@xhmikosr/decompress-tarbz2": "^8.0.2",
-    "@xhmikosr/decompress-targz": "^8.0.1",
-    "@xhmikosr/decompress-unzip": "^7.0.0",
-    "@xhmikosr/downloader": "^15.0.2",
-    "chalk": "^5.4.1",
+    "@felipecrs/decompress-tarxz": "^5.0.4",
+    "@xhmikosr/decompress-tar": "^8.1.0",
+    "@xhmikosr/decompress-tarbz2": "^8.1.0",
+    "@xhmikosr/decompress-targz": "^8.1.0",
+    "@xhmikosr/decompress-unzip": "^7.1.0",
+    "@xhmikosr/downloader": "^15.2.0",
+    "chalk": "^5.5.0",
     "clipanion": "^4.0.0-rc.4",
     "cosmiconfig": "^9.0.0",
     "execa": "^9.6.0",
-    "listr2": "^9.0.0",
+    "listr2": "^9.0.1",
     "rimraf": "^6.0.1"
```

Comment:
Bumping in package.json unnecessarily means packages that install bindl as dependency will potentially have to install more dependencies.

For example, let's say package-a depends on chalk `~5.4.0`:

Before this PR, npm would only install one copy of chalk (`5.4.1`), which satisfies both package-a and bindl needs.

After this PR, npm would install both `5.4.1` to satisfy package-a, and chalk `5.5.0` to satisfy bindl.

So... not sure if this is a good idea.

Reply:
That is not true... npm deduplicates the dependencies like all package managers.